File: Divyansh Singh/Array/Majority Element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int count=0,majority=0;
        for(auto i:nums){
            if(count==0){
                majority=i;
                count=1;
            }
            else if(i==majority){
                count++;
            }
            else{
                count--;
            }
        }
        return majority;
    }
};

File: Khushboo Singh/Array/Majority Element.java
class Solution {
    public int majorityElement(int[] nums) {
        int c=0,m=0;
        for(int num:nums){
            if(c==0){
                m=num;
                c++;

            }
            else if(m==num){

                c++;
            }
            else{
                c--;
            }
        
        }
        return m;
        
    }
}
